app/javascript/controllers/postal_code_controller.js

```diff
@@ -30,10 +30,11 @@ export default class extends Controller {
         const cardsItems = document.querySelector("#cards-items");
         cardsItems.innerHTML='';
 
+        // This needs to be updated
         items.forEach((item) => {
           console.log(item)
           const itemCard = `<div class="card-product" data-id="${item[0].id}">
-              <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/skateboard.jpg" />
+              <img src="https://images.unsplash.com/photo-1554136383-fa88b2d86aaf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=928&q=80" />
```

Comment:
I accidentally changed this and tried to change it back. Feel free to change this back to anything you want after.